plugins/obs-webrtc/webrtc/build.rs

```diff
@@ -0,0 +1,17 @@
+use std::{env, path::PathBuf};
+
+fn main() {
+    let crate_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
```

Comment:
Is there a better way to pass this information to the code than with environment variables (e.g. via a compiler parameter)? Environment variables are brittle and allow for unexpected behaviour via user interaction, something I'd like to avoid.

Also just for my understanding, is this a Rust program that will be compiled just for the host to generate the C-bindings (so the fact that this directory is HOST-specific is not an issue for the shipped product)?

Reply:
Correct, this is just preamble build logic that runs to generate the header.

Reply:
Yeah I'm fine with it if it's just the way it works - I try to avoid such approaches as best as possible, but you gotta do what you gotta do. 😅